lib/views/sensor_widget.dart

```diff
@@ -25,7 +25,7 @@ class SensorWidget extends StatelessWidget {
           ..color = Colors.black54
           ..center,
           AutoSizeText(
-            "${sensor?.value.toStringAsFixed(2) ?? "N/A"} ${sensor?.unit}",
+            "${sensor?.value.toStringAsFixed(2) ?? "N/A"} ${sensor?.unit.replaceAll("deg.C", "°C")}",
```

Comment:
should be a safe call, if we start with a safe call:
```suggestion
            "${sensor?.value?.toStringAsFixed(2) ?? "N/A"} ${sensor?.unit?.replaceAll("deg.C", "°C")}",
```